framework_tvb/tvb/interfaces/web/static/style/base.css

```diff
@@ -4,7 +4,7 @@
  * TheVirtualBrain-Scientific Package (for simulators). See content of the
```

Comment:
CSS files seem to have the same problem (not new in this commit, but from before) like the JS files.
They are clearly "framework" related, not "simulator"

Reply:
They are okay now.